CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Ability to enable multiplex sync (`Realm.Sync.enableSessionMultiplexing()`)
```

Comment:
This was available previously so this is better to be moved to `Fixes` since it fixes the TypeScript definition and adds a documentation. 

Maybe like so: Fixed TypeScript definitions and added documentation for `Realm.Sync.enableSessionMultiplexing()`